includes/models/model.llms.section.php

```diff
@@ -14,25 +14,44 @@
  * LLMS_Section model class
  *
  * @since 1.0.0
- * @since 3.24.0 Unknown.
  * @since 4.0.0 Remove deprecated class methods.
+ *
+ * @property int    $order         The section's order within its parent course.
+ * @property int    $parent_course The WP_Post ID of the section's parent course.
+ * @property string $title         The title / display name of the section.
```

Comment:
not sure about this.. the title is something inherited from the abstract model (it's actually a wp post property).
If we look at the other extending post models looks like we only document the custom (meta) properties...

Reply:
I got yelled at by a developer the other day on Slack for not documenting things well enough so I decided I was going to start documenting the inherited properties too reduce confusion for some

Reply:
Ahah I see :)